packages/ui/src/components/dialog.tsx

```diff
@@ -46,7 +47,7 @@ const DialogContent = React.forwardRef<React.ElementRef<typeof DialogPrimitive.C
     })
 
     return (
-      <DialogPortal>
+      <DialogPortal container={getGitnessShadowRoot()}>
```

Comment:
we need to find a way to avoid hardcoding a reference to gitness in `ui` package